msisdn-gateway/e2e-test/test/control-script.py

```diff
@@ -11,63 +11,66 @@
 
 host = "https://msisdn.services.mozilla.com"
 
+
 def clean_up_message_logs(client):
-	# clean up the previous message logs
-	messages = client.messages.list()
-	for i in range(0, len(messages)):
-		client.messages.delete(messages[i].sid)
+    # clean up the previous message logs
+    messages = client.messages.list()
+    for i in range(0, len(messages)):
+        client.messages.delete(messages[i].sid)
+
```

Comment:
Most of this nonsense was just converting tabs to 4 spaces to appease the lint gods.
You can omit whitespace diffs using this URL: https://github.com/mozilla-services/services-test/pull/41/files?w=0